packages/webapp/components/layouts/AccountLayout/AccountPageContainer.tsx

```diff
@@ -63,8 +63,8 @@ export const AccountPageContainer = ({
         className={classNames(
           'h-full overflow-y-scroll',
           footer
-            ? 'min-h-[calc(100vh-11.25rem)] max-h-[calc(100vh-11.25rem)]'
-            : 'min-h-[calc(100vh-7.25rem)] max-h-[calc(100vh-7.25rem)]',
+            ? '!min-h-[calc(100dvh-11.25rem)] !max-h-[calc(100dvh-11.25rem)] min-h-[calc(100vh-11.25rem)] max-h-[calc(100vh-11.25rem)]'
```

Comment:
Had to use `!important` because for some reason in production `vh` classes get priority. Not sure why.

Reply:
Should not matter because if browser does not support `dvh` it will take `vh` styles no matter the `!important`.